### PR TITLE
[INLONG-4405][Sort] Support run SQL script

### LINKED ISSUE #4405: [Feature] sort support run sql script
### Description

use sql script directly to run a flink job.

### Use case

sort core read sql script file.
#### sql script file
```sql
  CREATE TABLE `table_1`(
    `age` INT,
    `name` STRING)
    WITH (
    'connector' = 'mysql-cdc-inlong',
    'hostname' = 'localhost',
    'username' = 'root',
    'password' = 'inlong',
    'database-name' = 'test',
    'scan.incremental.snapshot.enabled' = 'false',
    'server-time-zone' = 'GMT+8',
    'table-name' = 'user'
);
CREATE TABLE `table_2`(
    PRIMARY KEY (`name`,`age`) NOT ENFORCED,
    `name` STRING,
    `age` INT)
    WITH (
    'connector' = 'jdbc',
    'url' = 'jdbc:postgresql://localhost:5432/postgres',
    'username' = 'postgres',
    'password' = 'inlong',
    'table-name' = 'public.user',
   'port' = '3306'
);
INSERT INTO `table_2` 
    SELECT 
    `name` AS `name`,
    `age` AS `age`
    FROM `table_1`;
```
#### use flink run sort and sql script
```shell
./bin/flink run -c org.apache.inlong.sort.Entrance inlong-sort/sort-[version].jar \
--sql.script.file /YOUR_SQL_SCRIPT_DIR/mysql-to-postgres.sql
```

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by gong**
### Title Name: [INLONG-4405][Sort] Support run SQL script

Fixes #4405 

### Motivation

sort support run sql script.

### Modifications

Add `NativeFlinkSqlParser` to parse SQL file.
In `Entrance` class, run SQL script by `--sql.script.file` configuration.

### CODE REVIEW

**Review by healchow — APPROVED** [review]

**Comment by dockerzhang** [comment]
here we need a document guide and example to show how to use it.

File: inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/parser/impl/NativeFlinkSqlParser.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    @Override
+    public ParseResult parse() {
+        Preconditions.checkNotNull(statements, "sql statement set is null");
+        Preconditions.checkNotNull(tableEnv, "tableEnv is null");
+        List<String> createTableSqls = new ArrayList<>();
+        List<String> insertSqls = new ArrayList<>();
+        String[] statementSet = statements.split(";");
+        for (String statement : statementSet) {
+            statement = statement.trim();
+            if (statement.startsWith("CREATE TABLE") || statement.startsWith("create table")) {
```
Comment (yunqingmoswu) [inline inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/parser/impl/NativeFlinkSqlParser.java:83]: It is recommended to ignore case when comparing strings.

File: inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/parser/impl/NativeFlinkSqlParser.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                createTableSqls.add(statement);
+            } else if (statement.startsWith("INSERT INTO") || statement.startsWith("insert into")) {
```
Comment (yunqingmoswu) [inline inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/parser/impl/NativeFlinkSqlParser.java:85]: It is recommended to ignore case when comparing strings.

File: inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/parser/impl/NativeFlinkSqlParser.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+     *
+     * @return ParseResult the result of parsing
+     */
+    @Override
+    public ParseResult parse() {
+        Preconditions.checkNotNull(statements, "sql statement set is null");
+        Preconditions.checkNotNull(tableEnv, "tableEnv is null");
+        List<String> createTableSqls = new ArrayList<>();
+        List<String> insertSqls = new ArrayList<>();
+        String[] statementSet = statements.split(";");
```
Comment (GanfengTan) [inline inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/parser/impl/NativeFlinkSqlParser.java:80]: Only need to support DDL, DML? Do we need to consider DQL?

Comment (gong) [reply inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/parser/impl/NativeFlinkSqlParser.java:80]: > 

json to sql just need to execute `create table` and `insert into`. So here sql keep consistent.

**Comment by gong** [comment]
> here we need a document guide and example to show how to use it.

please see the issue ：[4405](https://github.com/apache/incubator-inlong/issues/4405).
we can add document guide in inlong-website in the future.

**Comment by gong** [comment]
> here we need a document guide and example to show how to use it.

@dockerzhang please see the issue ：[4405](https://github.com/apache/incubator-inlong/issues/4405).
we can add document guide in inlong-website in the future.

File: inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/parser/impl/NativeFlinkSqlParser.java (hunk 1 above)
Comment (gong) [reply inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/parser/impl/NativeFlinkSqlParser.java]: > It is recommended to ignore case when comparing strings.

done

File: inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/parser/impl/NativeFlinkSqlParser.java (hunk 2 above)
Comment (gong) [reply inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/parser/impl/NativeFlinkSqlParser.java]: > It is recommended to ignore case when comparing strings.

done

**Review by GanfengTan — APPROVED** [review]

**Pull request merged**